Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -584,6 +584,12 @@ def audit_homepage
       problem "Please use https:// for #{homepage}"
     end
 
+    case homepage
+    when %r{^http://([^/]*)sourceforge\.net(/|$)}i,
+         %r{^http://([^/]*)sf\.net(/|$)}i
+      problem "#{homepage} should be `https://#{$1}sourceforge.io/`"
```

Comment:
Yes, makes sense. But in this case you should add an additional assertion for your test case such as it is done here: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/audit_test.rb#L456 because the tests are now failing.